tools/lint

```diff
@@ -111,13 +109,15 @@ def run():
     @linter_config.lint
     def custom_py():
         # type: () -> int
-        failed = check_custom_checks_py()
+        failed = python_rules.check(by_lang)
         return 1 if failed else 0
 
     @linter_config.lint
     def custom_nonpy():
         # type: () -> int
-        failed = check_custom_checks_nonpy()
+        failed = True
+        for rule in non_py_rules:
+            failed = failed & rule.check(by_lang)
         return 1 if failed else 0
```

Comment:
Can you explain this change?  It doesn't super make sense to me, with `failed` starting at True.  I think you want the opposite starting at False and changing it to True if any individual rule fails.

Reply:
I have changed it now. Thanks for pointing out.